src/test/java/org/example/AddRemoveItem.java

```diff
@@ -1,7 +1,109 @@
 package org.example;
 
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
 
 public class AddRemoveItem {
 
+    private WebDriver webDriver;
+
+    @BeforeMethod
+    public void setUpDrive() {
```

Comment:
Typo in method name : missing an 'r' at the end.

Reply:
Added the "r" at the end of the method name.